Configuration/Settings.yaml

```diff
@@ -86,6 +86,7 @@ Neos:
             contentCanvas:
               src: '${Neos.Ui.NodeInfo.uri(documentNode, controllerContext)}'
               contextPath: '${q(documentNode).property(''_contextPath'')}'
+              backgroundColor: '#ffffff'
```

Comment:
Hey, we keep only those things that will be changing on runtime inside of state. I think this should go to some other path in settings, that should be exposed here https://github.com/neos/neos-ui/blob/master/Resources/Private/Fusion/Backend/Root.fusion#L13

Reply:
ack, and furthermore the "*state" part of the settings is private (e.g. only for unplanned extensibility).

So really cool feature, just we need to move the settings key :)

Reply:
working on it 👍